src/components/SocialLocationPreview/SocialLocationPreview.style.tsx

```diff
@@ -7,7 +7,9 @@ export const Wrapper = styled.div<{
   border?: Boolean;
 }>`
   box-shadow: ${props =>
-    props.border ? 'none' : '0px 12px 40px rgba(0, 0, 0, 0.2)'};
+    props.noShadow || props.border
```

Comment:
Should there be parens around this?  As-is, I think you'll end up with `box-shadow: true` if `props.noShadow` is true? But I could easily be confused.

Reply:
Now you've confused me and I'm not sure but parens wouldn't hurt so I'll add them.

Reply:
Looks like you didn't add them, which might mean that I was confused and `prettier` removed them because they're not necessary. 🤷

Reply:
yeah I added them and prettier re-removed them so :shrug: